src/presentation/http/router/user.test.ts

```diff
@@ -0,0 +1,46 @@
+import { describe, test, expect } from 'vitest';
+
+
+describe('user API', () => {
+  describe('GET /user/myself', () => {
+    test('returns user with status code 200 if user exists', async () => {
+      const expectedStatus = 200;
+      const userId = 1;
+      const accessToken = global.auth(userId);
+      const expectedUser =   {
+        'id': '1',
+        'email': 'a@a.com',
+        'name': 'Test user 1',
+        'photo': '',
+      };
+      const response = await global.api?.fakeRequest({
+        method: 'GET',
+        headers: {
+          authorization: `Bearer ${accessToken}`,
+        },
+        url: '/user/myself',
+      });
+
+      expect(response?.statusCode).toBe(expectedStatus);
+
+      const body = response?.json();
+
+      expect(body).toStrictEqual(expectedUser);
+    });
+    test('returns you must be authorized response if user is not authorized', async () => {
```

Comment:
maybe rephrase the test title? something like `Returns response with status 401 when user is not authorized`